src/app/choose-us/choose-us.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="rental-container">
```

Comment:
classes are too long, please try to break it down...

Reply:
at the max, it should go till 5 words @tsehringyangden

Reply:
resolved..................thank you